### ISSUE TITLE
Need Help

### ISSUE THREAD

**Issue by Red09discord** [Jisagi/Discord-guild-copy#56]
I need to speak with you in private can i have your discord support server link?

**Comment by Jisagi** [comment]
What is it, you cannot ask here? There is no "discord support server" btw

**Comment by Red09discord** [comment]
Ok, actually I want to clone the server with a self bot and I want to BUILD the server with a bot token so that it's not against tos pls help

**Comment by Red09discord** [comment]
Or is there any way for me to only clone server roles channels etc. Without actually cloning it in another server.. just get the server roles perms etc in a js or py file and then use a bot token and select the file so that it loads the cloned server into a new one plsssss

**Comment by Jisagi** [comment]
I'm just gonna copy what I wrote in another issue:

1. The discord.js (d.js) library doesn't even support user tokens anymore, which means, they just don't work at all, even if you try. d.js doesn't know how to handle user tokens (anymore)
2. Older versions of d.js would still support user tokens, BUT those don't support the stuff I do with the script, so they don't work here either
3. Running a bot with a user token is already against their api. It doesn't matter if they can't detect it because you don't create lets say a channel or just get scrape the data.

TL;DR: No, doesn't work, the d.js library doesn't work with a user token
